tests/fixtures/config-extends/package3/subdir/foo.js

```diff
@@ -1 +0,0 @@
-var a = 1;
```

Comment:
This package was copied from package2.. this file was not being used in any test.